.github/workflows/mega-linter.yml

```diff
@@ -44,6 +44,15 @@ jobs:
           node-version: 14.x
       - run: yarn install --frozen-lockfile
 
+      # Check if all i18n terms in code are synced with source (English) translation file.
+      - name: Check English translation is synced
```

Comment:
Update since https://github.com/oasisprotocol/oasis-wallet-web/pull/753 + https://github.com/oasisprotocol/oasis-wallet-web/pull/799: translation paths are now typechecked, so typescript should already complain about any translation paths missing from en.json

Reply:
this additionally ensures that the strings match